src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationOptions.cs

```diff
@@ -56,6 +56,11 @@ public string DisplayName
         /// </summary>
         public bool SaveTokensAsClaims { get; set; }
 
+        /// <summary>
+        /// Gets or sets the lifetime of the correlation cookie used to prevent cross-site scripting attacks.
```

Comment:
Yes, but what this really means is that this is the time limit for completing the authentication flow.